src/main/java/uk/gov/di/cri/experian/kbv/api/gateway/SAARequestMapper.java

```diff
@@ -92,9 +92,7 @@ private void setLocationDetails(SAARequest saaRequest, PersonIdentity personIden
 
         if (personIdentity.getAddresses() != null && personIdentity.getAddresses().size() > 0) {
             ukLocation.setPostcode(personIdentity.getAddresses().get(0).getPostcode());
-            ukLocation.setDistrict(personIdentity.getAddresses().get(0).getTownCity());
```

Comment:
There is `District` in `personIdentity` looking at the paas app, under `uklocation` we also have flat, houseName, houseNumber